src/icon-manifest/supportedExtensions.ts

```diff
@@ -634,6 +634,7 @@ export const extensions: IFileCollection = {
     { icon: 'wercker', extensions: ['wercker.yml'], filename: true, format: FileFormat.svg },
     { icon: 'wolfram', extensions: [], languages: [languages.wolfram], format: FileFormat.svg },
     { icon: 'word', extensions: ['doc', 'docx', 'docm', 'dot', 'dotx', 'dotm', 'wll'], format: FileFormat.svg },
+    { icon: 'wurst', extensions: ['wurst'], languages: [languages.wurstlang], format: FileFormat.svg },
```

Comment:
In this case you don't need to specify the `extensions` value as then you'll be not leveraging the language id flexibility.